linera-core/src/client/signer.rs

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::BTreeMap;
+
+use linera_base::{
+    crypto::{AccountPublicKey, AccountSecretKey, AccountSignature, CryptoHash, CryptoRng, Signer},
+    identifiers::AccountOwner,
+};
+use serde::{Deserialize, Serialize};
+
+/// In-memory signer.
+pub struct InMemSigner {
```

Comment:
How did this end up in `linera-core` while the trait is in `linera-base`? :thinking: I'd expect these to be next to each other.

Reply:
The reason I didn't do it initially was that it requires lots of feature-flagging the `CryptoRng` usage b/c `linera-base` is dependency for web crates.

I will move it back and add those feature flags though.

Reply:
Done

Reply:
`linera-core` is also a dependency for Web crates :) (but not on-chain builds)